pytorch_lightning/callbacks/gradient_accumulation_scheduler.py

```diff
@@ -1,5 +1,5 @@
 r"""
-Gradient Accumulator
+GradientAccumulator
```

Comment:
why together? it is not even linked to a class, isn't it?

Reply:
yeah, it should be separate

Reply:
ok, there are a few more...